packages/spacecat-shared-rum-api-client/README.md

```diff
@@ -298,6 +298,43 @@ Calculates the amount of non-inorganic (earned and owned) traffic and the click-
 
 ```
 
+### formVitals
+
+Checks the form vitals for a given domain within the requested interval. It checks whether the
+URL has forms embedded in it or not.
```

Comment:
Looks like you're collecting the forms viewed/engaged/submitted information, rather than checking, can you please elaborate the description.

It'll be great, if you can add, how you're planning to leverage this information for obtaining useful results, like you'd like to check the forms that have higher viewed but less engagement, high errors etc.

Reply:
done